ui/app/components/auth/form-template.ts

```diff
@@ -157,63 +189,7 @@ export default class AuthFormTemplate extends Component<Args> {
   }
 
   @action
-  handleNamespaceUpdate(event: HTMLElementEvent<HTMLInputElement>) {
-    // update query param
+  async handleNamespaceUpdate(event: HTMLElementEvent<HTMLInputElement>) {
     this.args.handleNamespaceUpdate(event.target.value);
-    // reset tabs
-    this.authTabs = null;
-    // fetch mounts for that namespace
-    this.fetchMounts.perform(500);
```

Comment:
no longer necessary since the namespace update re-fires the model hook which re-requests mounts 🎉